src/components/CookForm/CookForm.js

```diff
@@ -13,7 +13,7 @@ import styles from './styles.scss';
   {
     form: 'cookForm',
     fields: ['id', 'main_photo_temp_image_id', 'other_photos_temp_image_ids', 'removing_other_photos', 'removing_sanitary_book_photos',
-      'sanitary_book_photos', 'sanitary_book_photos_temp_image_ids', 'first_name', 'last_name', 'other_photos', 'main_photo', 'location_attributes', 'location',
+      'sanitary_book_photos', 'sanitary_book_photos_temp_image_ids', 'first_name', 'last_name', 'first_name_genitive', 'last_name_genitive', 'full_name_genitive', 'other_photos', 'main_photo', 'location_attributes', 'location',
```

Comment:
я, принципе его только и использовал. вместо двух(first + last) использовал full.
> 18 апр. 2016 г., в 14:19, Rodion <notifications@github.com> написал(а):
> 
> In src/components/CookForm/CookForm.js <https://github.com/sars/appetini-front/pull/28#discussion_r60041468>:
> 
> > @@ -13,7 +13,7 @@ import styles from './styles.scss';
> >    {
> >      form: 'cookForm',
> >      fields: ['id', 'main_photo_temp_image_id', 'other_photos_temp_image_ids', 'removing_other_photos', 'removing_sanitary_book_photos',
> > -      'sanitary_book_photos', 'sanitary_book_photos_temp_image_ids', 'first_name', 'last_name', 'other_photos', 'main_photo', 'location_attributes', 'location',
> > +      'sanitary_book_photos', 'sanitary_book_photos_temp_image_ids', 'first_name', 'last_name', 'first_name_genitive', 'last_name_genitive', 'full_name_genitive', 'other_photos', 'main_photo', 'location_attributes', 'location',
> зачем full_name_genitive?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/sars/appetini-front/pull/28/files/1466e0fe003b339f2577c5f774caf0dd296db985#r60041468>